src/main.rs

```diff
@@ -222,7 +222,7 @@ pub struct TopLevel {
     vdpa: Vec<String>,
 
     #[argh(option, long = "vsock")]
-    /// cid=<context_id>, socket=<socket_path>, iommu=on|off, id=<device_id>, pci_segment=<segment_id>
+    /// cid=<context_id>, socket=<socket_path>, iommu=on|off, id=<device_id>, pci_segment=<segment_id>, vsock_mode=bidirectional|host2guest|guest2host
```

Comment:
This is a bit verbose — what about `vsock_listen=host,guest` (where you could also specify just one of the two), or something?

Reply:
Thx a lot for your review!
However, I prefer `vsock_mode` a little, 'cause this is closer to what is described in the vsock.md. Also, there is already a `VhostMode` :-)